<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timetable</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>1001</width>
      <height>631</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255)</string>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab_0">
     <attribute name="title">
      <string>Понедельник</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>531</width>
        <height>231</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Аудитория</string>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Предмет</string>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Преподаватель</string>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Начало</string>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>8</pointsize>
        </font>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <attribute name="title">
      <string>Вторник</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>531</width>
        <height>231</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Аудитория</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Предмет</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Преподаватель</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Начало</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Среда</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>531</width>
        <height>231</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Аудитория</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Предмет</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Преподаватель</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Начало</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Четверг</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>531</width>
        <height>231</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Аудитория</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Предмет</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Преподаватель</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Начало</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Пятница</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_5">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>531</width>
        <height>231</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Аудитория</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Предмет</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Преподаватель</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Начало</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Суббота</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget_6">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>531</width>
        <height>231</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>1.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5.</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
        </font>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Аудитория</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Предмет</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Преподаватель</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Начало</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Обновить</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
